includes/zigbee2mqtt/configuration.yaml

```diff
@@ -0,0 +1,21 @@
+frontend:
+  port: 8081
+homeassistant: false
+permit_join: true
+mqtt:
+  base_topic: zigbee2mqtt
+  server: mqtt://localhost
+  user: openhabian
```

Comment:
Note users can change username ("username" in openhabian.conf), please enable your code for this

Reply:
All right. I changed that. 
And I changed that the update-function is chosen whenever a config file of zigbee2mqtt ist found. This prevents that a config gets overwritten.

Reply:
fixed a bug with default user "openhabian".